src/fetchAndTransformArticle.js

```diff
@@ -2,12 +2,23 @@ import cheerio from 'cheerio';
 import { fetchArticle } from './api/articleApi';
 import { transform } from './transformers';
 
-export async function transformArticle(article, lang, accessToken) {
+export async function transformArticle(
+  article,
+  lang,
+  accessToken,
+  isOembedRequest
```

Comment:
How about just sending an object named options instead of isOembedRequest. 

I also think isOembedRequest is bad name. It would be better to describe what the option actually does (i.e. removeRelatedContent).

Reply:
We should probably add lang and accessToken to the option object, but we can fix that later.